listenbrainz/webserver/rabbitmq_connection.py

```diff
@@ -1,4 +1,3 @@
-import sys
 import queue
 from time import sleep
 import pika
```

Comment:
pika?

Reply:
`pika` is still used, we removed `pika_pool`

Reply:
Oh, right. Thanks.